File: 250323/정사각 알파벳 출력하기/print-alpabet-in-square.cpp
#include <iostream>
using namespace std;

int main()
{
    int n = 0, alpha = 65;
    cin >> n;
    for(int i = 1; i <= n; i++) {
        for(int j = 1; j <= n; j++) {
            if(alpha > 90) {
                alpha = 65;
            }
            cout << (char)alpha;
            alpha++;
        }
        cout << "\n";
    }
    return 0;
}

File: 250323/정사각 알파벳 출력하기/print-alpabet-in-square.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();
        int alpha = 65;
        for(int i = 1; i <= n; i++) {
            for(int j = 1; j <= n; j++) {
                System.out.print((char)alpha);
                alpha++;
            }
            System.out.println();
        }
        sc.close();
    }
}